src/settings/components/CollectionActions.jsx

```diff
@@ -45,7 +45,7 @@ const CollectionActions = ({
     setMenuAnchorEl(null);
   };
 
-  const handleRemoveResult = (removed) => {
+  const defaultHandleRemoveResult = (removed) => {
     setRemoving(false);
```

Comment:
Your new method doesn't call `setRemoving`.

Reply:
Oh, you're right. Seems to work out alright though, I think it's supposed to hide the popup? Anyways, I'll apply your suggestion from below to fix this.